localstack/mock/infra.py

```diff
@@ -35,6 +36,8 @@
 INSTALL_DIR_ES = '%s/elasticsearch' % INSTALL_DIR_INFRA
 TMP_ARCHIVE_ES = '/tmp/localstack.es.zip'
 
+ERROR_PROBABILITY = 0.05
```

Comment:
Couldn't we be using `ERROR_PROBABILITY` in constants.py (or config.py) instead of `KINESIS_RETURN_ERRORS`? `ERROR_PROBABILITY == 1` would have the same effect as `KINESIS_RETURN_ERRORS == True`